woocommerce/woocommerce-android
# [Tablet Orders] Keep the drawer always visible

Closes: #10473
Closes: #10470


### Description

* Shows minimised version of the drawer always visible even when there is nothing in the order
* Removes "order total" (behind FF)
* Loading indicated in the top for both Edit and Creation modes (NOT behind FF)

### Testing instructions

* Make sure that Loading indicator in the top doesn't break anything to the current flow
* Notice in the debug mode new, minimized bottom sheet always visible and extended when there is something in the order

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/5395530c-676f-47ec-8fe9-91a7d85dc28a
